reframe/core/schedulers/slurm.py

```diff
@@ -227,7 +230,21 @@ def emit_preamble(self, job):
 
     def submit(self, job):
         cmd = f'sbatch {job.script_filename}'
-        completed = _run_strict(cmd, timeout=self._submit_timeout)
+        intervals = itertools.cycle([1, 2, 3])
+        while True:
+            try:
+                completed = _run_strict(cmd, timeout=self._submit_timeout)
+                break
+            except SpawnedProcessError as e:
```

Comment:
```suggestion
            except SpawnedProcessError:
```

Reply:
Can I have access to `e.stderr` without it?

Reply:
Ok, I missed that. But then don't `raise e`, do just `raise`.